core/src/migrations/000067-addExportProcessors.ts

```diff
@@ -0,0 +1,97 @@
+export default {
+  up: async function (migration, DataTypes) {
+    await migration.sequelize.transaction(async () => {
+      await migration.createTable("exportProcessors", {
+        id: {
+          type: DataTypes.STRING(191),
+          primaryKey: true,
+        },
+
+        state: {
+          type: DataTypes.STRING(191),
+          allowNull: false,
+        },
+
+        remoteKey: {
+          type: DataTypes.STRING(191),
+          allowNull: true,
+          defaultValue: null,
+        },
```

Comment:
Could this get longer than 191?  What if the remoteKey has a bunch of UUIDs or similar in it?

Reply:
changed to TEXT type